### DISCUSSION TITLE
World portals broken

### DISCUSSION THREAD

**Discussion by Amogos69** [xoxor4d/p2-rtx#25] [General]
I once unistalled the mod and when I use the portals they appear closed and all the world portals are completely broken. not even reinstalling or deleting all saved data would work. can someone please help fix this?

https://github.com/user-attachments/assets/2af20fd9-f5c1-4882-b3c2-f07af106ffb1

**Comment by xoxor4d** [comment]
> I once unistalled the mod and when I use the portals they appear closed and all the world portals are completely broken. not even reinstalling or deleting all saved data would work. can someone please help fix this?

See Issue https://github.com/xoxor4d/p2-rtx/issues/10
